### DISCUSSION TITLE
Problem while checking the parameters

### DISCUSSION THREAD

**Discussion by maxlelyonais** [mrdbourke/pytorch-deep-learning#226] [Q&A]
Good Morning/Evening/Night.

This error appeared while I was doing the course. I don't know if I mispelled something or it is from the program.

This is the code:

# Create a random seed
torch.manual_seed(42)

#Create an instance of the model ( this is a subclass of nn.Module)

model_0 = LinearRegressionModel()

#Check out the parameters
model_0.parameters()

And this is the error:

TypeError                                 Traceback (most recent call last)
[<ipython-input-24-dcfaa8f2db3a>](https://localhost:8080/#) in <module>
      4 #Create an instance of the model ( this is a subclass of nn.Module)
      5 
----> 6 model_0 = LinearRegressionModel()
      7 
      8 #Check out the parameters

[<ipython-input-12-0d652b9d3fc2>](https://localhost:8080/#) in __init__(self)
      9                                             requires_grad=True,
     10                                             dtype=torch.float)) ## Create a weight parameter
---> 11     self.bias = nn.Parameter(torch.randn(1,
     12                                          required_grad = True,
     13                                          dtype = torch.float))

TypeError: randn() received an invalid combination of arguments - got (int, dtype=torch.dtype, required_grad=bool), but expected one of:
 * (tuple of ints size, *, torch.Generator generator, tuple of names names, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (tuple of ints size, *, torch.Generator generator, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (tuple of ints size, *, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (tuple of ints size, *, tuple of names names, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)

**Comment by AlessandroMiola** [comment]
Seems you mispelled `requires_grad` (you wrote `required_grad` instead).
